spring-cloud-dataflow-server-kubernetes-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -471,19 +528,37 @@ Successfully targeted http://130.211.203.246/
 dataflow:>
 ```
 +
-. Register the Docker with Rabbit binder versions of the `time` and `log` apps using the shell.
+. Register the Docker binder versions of the `time` and `log` apps using the shell.
```

Comment:
"Register the Docker binder versions" -> "Register the Docker images of the Rabbit binder based `time` and `log` apps using the shell"